azure-mgmt-containerregistry/src/main/java/com/azure/management/containerregistry/implementation/RegistryEncodedTaskStepImpl.java

```diff
@@ -76,7 +75,7 @@ public RegistryEncodedTaskStepImpl withBase64EncodedTaskContent(String encodedTa
     @Override
     public RegistryEncodedTaskStepImpl withBase64EncodedValueContent(String encodedValueContent) {
         if (isInCreateMode()) {
-            this.inner.withBase64EncodedValuesContent(encodedValueContent);
+            this.inner.withEncodedValuesContent(encodedValueContent);
```

Comment:
Is it same functionality before/after?

Reply:
Yes, the encoded values is always base64 encode. This difference is due to generation.